spec/rubocop/cop/rspec/overwriting_setup_spec.rb

```diff
@@ -55,7 +55,7 @@
     RUBY
   end
 
-  it 'handles unnamed subjects' do
+  it 'registers an offense for unnamed subjects' do
```

Comment:
This is not correct. I believe it's more like "registers an offense for overriding an unnamed subject".